{
	"document_title": "Security update for python-django-grappelli",
	"document_type": "SUSE Patch",
	"document_publisher": {
		"type": "Vendor",
		"contact_details": "security@suse.de",
		"issuing_authority": "SUSE Security Team"
	},
	"document_tracking": {
		"identification": {
			"id": "openSUSE-SU-2023:0384-1"
		},
		"status": "Final",
		"version": "1",
		"revision_history": {
			"revision": {
				"number": "1",
				"date": "2023-11-30T10:47:42Z",
				"description": "current"
			}
		},
		"initial_release_date": "2023-11-30T10:47:42Z",
		"current_release_date": "2023-11-30T10:47:42Z",
		"generator": {
			"engine": "cve-database/bin/generate-cvrf.pl",
			"date": "2017-02-24T01:00:00Z"
		}
	},
	"document_notes": [
		{
			"text": "Security update for python-django-grappelli",
			"title": "Topic",
			"type": "Summary"
		},
		{
			"text": "This update for python-django-grappelli fixes the following issues:\n\nUpdate to 2.14.4:\n\n- CVE-2021-46898: Fixed views/switch.py vulnerable to protocol-relative URL attacks (boo#1216481)\n- Fixed: Redirect with switch user.\n- Improved: Remove extra filtering in AutocompleteLookup.\n- Improved: Added import statement with URLs for quickstart docs.\n- Improved: Added additional blocks with inlines to allow override.\n- Fixed: Compatibility with Django 3.1.\n- Fixed: Docs about adding Grappelli documentation URLS.\n",
			"title": "Details",
			"type": "General"
		},
		{
			"text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
			"title": "Terms of Use",
			"type": "Legal Disclaimer"
		},
		{
			"text": "openSUSE-2023-384",
			"title": "Patchnames",
			"type": "Details"
		}
	],
	"documentdistribution": "Copyright SUSE LLC under the Creative Commons License 4.0 with Attribution (CC-BY-4.0)",
	"document_references": [
		{
			"type": "Self",
			"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/FVC472DTXM3I3SVFJZ3UKZVVBMB6XYMZ/",
			"description": "E-Mail link for openSUSE-SU-2023:0384-1"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/support/security/rating/",
			"description": "SUSE Security Ratings"
		},
		{
			"type": "Self",
			"url": "https://bugzilla.suse.com/1216481",
			"description": "SUSE Bug 1216481"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/security/cve/CVE-2021-46898/",
			"description": "SUSE CVE CVE-2021-46898 page"
		}
	],
	"product_tree": {
		"branch": [
			{
				"type": "Product Family",
				"name": "SUSE Package Hub 15 SP4",
				"branch": [
					{
						"type": "Product Name",
						"name": "SUSE Package Hub 15 SP4",
						"full_product_name": {
							"text": "SUSE Package Hub 15 SP4",
							"product_id": "SUSE Package Hub 15 SP4"
						}
					}
				],
				"full_product_name": {}
			},
			{
				"type": "Product Family",
				"name": "openSUSE Leap 15.4",
				"branch": [
					{
						"type": "Product Name",
						"name": "openSUSE Leap 15.4",
						"full_product_name": {
							"text": "openSUSE Leap 15.4",
							"product_id": "openSUSE Leap 15.4",
							"cpe": "cpe:/o:opensuse:leap:15.4"
						}
					}
				],
				"full_product_name": {}
			},
			{
				"type": "Product Version",
				"name": "python3-django-grappelli-2.14.4-bp154.2.3.1",
				"full_product_name": {
					"text": "python3-django-grappelli-2.14.4-bp154.2.3.1",
					"product_id": "python3-django-grappelli-2.14.4-bp154.2.3.1"
				}
			}
		],
		"relationship": [
			{
				"product_reference": "python3-django-grappelli-2.14.4-bp154.2.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "SUSE Package Hub 15 SP4"
			},
			{
				"product_reference": "python3-django-grappelli-2.14.4-bp154.2.3.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Leap 15.4"
			}
		]
	},
	"vulnerability": [
		{
			"notes": [
				{
					"text": "views/switch.py in django-grappelli (aka Django Grappelli) before 2.15.2 attempts to prevent external redirection with startswith(\"/\") but this does not consider a protocol-relative URL (e.g., //example.com) attack.",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2021-46898",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"SUSE Package Hub 15 SP4:python3-django-grappelli-2.14.4-bp154.2.3.1",
						"openSUSE Leap 15.4:python3-django-grappelli-2.14.4-bp154.2.3.1"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "moderate"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {},
				"score_set_v_3": {
					"base_score_v_3": "6.1",
					"vector_v_3": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N"
				},
				"scoreset": {}
			},
			"remediations": [
				{
					"type": "Vendor Fix",
					"description": {
						"text": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
						"lang": "en"
					},
					"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/FVC472DTXM3I3SVFJZ3UKZVVBMB6XYMZ/"
				}
			],
			"references": [
				{
					"url": "https://www.suse.com/security/cve/CVE-2021-46898.html",
					"description": "CVE-2021-46898"
				},
				{
					"url": "https://bugzilla.suse.com/1216481",
					"description": "SUSE Bug 1216481"
				}
			]
		}
	]
}
